### PR TITLE
HA frontend change for alarm panel vacation mode

### PR DESCRIPTION
**Pull request by posixx**
## Proposed change

Add button and display of new vacation mode for alarm panels
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion: https://github.com/home-assistant/core/pull/45980 and https://github.com/home-assistant/architecture/discussions/500
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Waiting for backend

**Comment by frenck** [comment]
The backend is just approved.

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 1]
```diff
@@ -32,6 +32,7 @@ const ICONS = {
   armed_custom_bypass: "hass:security",
   armed_home: "hass:shield-home",
   armed_night: "hass:shield-home",
+  armed_vacation: "hass:shield-lock",
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts:35]:
```suggestion
  armed_vacation: "hass:shield-sun",
```

**Review by bramkragten — APPROVED** [review]

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts (hunk 1 above)
Comment (frenck) [reply src/panels/lovelace/cards/hui-alarm-panel-card.ts:35]: agree, that is better :)

**Comment by posixx** [comment]
@bramkragten could you look into the build error and fix?

**Comment by posixx** [comment]
Ok, frontend also ready for merge!

**Pull request merged**